src/components/Li.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Li = ({ todo, deleteTodo, toggleComplited }) => {
```

Comment:
this is the worst name you could give to the component 😄
ListItem is better or something like this